freqtrade/exchange/bittrex.py

```diff
@@ -98,16 +99,35 @@ def get_ticker(self, pair: str) -> dict:
             'last': float(data['result']['Last']),
         }
 
+	
     def get_ticker_history(self, pair: str, tick_interval: int) -> List[Dict]:
         if tick_interval == 1:
             interval = 'oneMin'
         elif tick_interval == 5:
             interval = 'fiveMin'
         else:
             raise ValueError('Cannot parse tick_interval: {}'.format(tick_interval))
-
-        data = _API_V2.get_candles(pair.replace('_', '-'), interval)
-
+        if pair in _cache.keys():
+                # pair is in cache retriev lastest candle
+                sdata = _API_V2.get_latest_candle(pair.replace('_', '-'), interval)
+                if not sdata.get('result'):
+                        raise ContentDecodingError('{message} params=({pair})'.format(
+                            message='Got invalid response from bittrex',
+                            pair=pair))
+                data = _cache[pair]
+                #this is the latest results we had
+                old_ticker = data['result'][-1];
+                #check timestamp is newer ...
+                if (sdata['result'][0]['T'] > old_ticker['T']) : 
+                        data['result'].append(sdata['result'][0])
+                elif (sdata['result'][0]['T'] == old_ticker['T']) :
+                        #if volume has changed, update the latest result with the new one
```

Comment:
I don't get the logic here? Why would we not update the ticker when the volume is decreasing?

Reply:
Volume is not only increasing for the same timestamp?

Reply:
Ah, so that was your logic. I'm guessing it should not change, because it should not return the ticker before it's complete.

Reply:
Well, it seems it is changing ...

Reply:
Okay :D I'm reaaaally starting to hate Bittrex API.

Reply:
Even if you change to any other platform, only relying on their data without checking save could lead to by trade decision. If an http proxy or CDN delivers you and old version of the data you need to ignore it